main.js

```diff
@@ -0,0 +1,55 @@
+/*task 6*/
+let user = 'John Doe';
+let student = 'Irina';
+
+console.log(user);
+console.log(student);
+
+user = student /*Irina*/
+
+console.log(user);
+
+/*task 7*/
+let test = 1;
+
+test++ /*2*/
+test = test + '1' /*21*/
+
+console.log(test);
+
+test = test - 1 /*0*/
+
+console.log(test);
+
+test = true; /*true*/
```

Comment:
It doesn't assign the task. You should convert the value of the variable `test` to boolean.

Reply:
Ok. I understood and corrected it.